src/App.js

```diff
@@ -19,36 +19,66 @@ import AddTodoForm from "./AddTodoForm";
-  const [todoList, setTodoList] = useState(
-    // Instructions: "Update the default state for `todoList` to read your `"savedTodoList"` item from `localStorage`. Then update your default state to parse the value of the `"savedTodoList"` item. Also had to add or statement || followed by empty bracket, so when the local storage is cleared & therefore empty, it still works.
-    JSON.parse(savedTodoListToParse) || []
-  );
+//   // Below sets new to do's. Creates a new state variable named `newTodo` with update function named `setNewTodo` (with useState hook)
 
-  // Instructions from 1-5: "Define a `useEffect` React hook with `todoList` as a dependency; inside the side-effect handler function, save the `todoList` inside `localStorage` with the key `"savedTodoList"` "
-  React.useEffect(() => {
-    localStorage.setItem("savedTodoList", JSON.stringify(todoList));
-  }, [todoList]);
+//   // const [newTodo, setNewTodo] = useState("");
 
-  // Instructions from 1-5: "Add a `return` statement in `useSemiPersistentState` that returns the `todoList` state variable and setter in an Array (just like how it's returned from the `useState` hook)"
-  return [todoList, setTodoList];
-};
+//   // Returns todoList state variable & state setter in an array (like how it's returned from a useState hook, Lesson 1-5)
+//   return [todoList, setTodoList];
+// };
 
 // ************************************************************************************************************************ //
 // APP COMPONENT / FUNCTION//////////////////////////////////////////////////////
 // ************************************************************************************************************************ //
 function App() {
-  //  "Update `App` to use new custom hook. Hint: Copy `useState` hook from before, but change `useState` to custom hook `useSemiPersistentState` (no arguments)" (Lesson 1-5)
-  const [todoList, setTodoList] = useSemiPersistentState("savedTodoList");
+  //  Commented out useSemiPersistentState b/c moved to async data. Previously had updated App to use it. Copied useState hook from before, but changde useState to useSemiPersistentState w/out any arguments. (Lesson 1-5)
+  // const [todoList, setTodoList] = useSemiPersistentState("savedTodoList");
+
+  // ************************************************************************************************************************ //
+  // STATE: Various situations dealing with State below
+
+  // Below stores ALL to do's. Creates new state variable called "todoList" w/"setter" called "setTodoList" w/an empty array as default value (was "todos" prior to that, but then had recommended to change to empty array). This was previously a part of "lifting state" step.
+  const [todoList, setTodoList] = useState(
```

Comment:
All good! Just one thing, in as much as you want to be having your exercise instructions as comments directly in your code for your easier understanding when you later wanna go through it, it really might cause you to create errors if you are not careful with it's placements. Something like the comment here in a State hook brackets.